mechanisms/terminator/micm/species.json

```diff
@@ -5,14 +5,51 @@
       "type": "CHEM_SPEC",
       "absolute tolerance": 1e-12,
       "molecular weight [kg mol-1]": 0.035453,
-      "__is advected": true
+      "__is advected": true,
+      "__is parameterized": false
```

Comment:
We just talked about this, but I’m again confused! MICM is going to solve this species, so should the parameterized option be set to true?

Reply:
No, never mind! I thought the other way